cmd/internal/shared/gatewayserver/config.go

```diff
@@ -32,6 +32,7 @@ var DefaultGatewayServerConfig = gatewayserver.Config{
 	FetchGatewayInterval:              10 * time.Minute,
 	FetchGatewayJitter:                0.2,
 	UpdateGatewayLocationDebounceTime: time.Hour,
+	UpdateConnectionStatsDebounceTime: 10 * time.Second,
```

Comment:
Isn't 10 seconds a lot considering some of the heavy duty gateways?

Reply:
At heart that's the point - reducing the writes that result due to these heavy duty gateways. But I think we can live with a 5 too.